llvm/lib/Target/EVM/EVMStackSolver.cpp

```diff
@@ -68,7 +59,7 @@ findStackTooDeep(const Stack &Source, const Stack &Target,
     return Result;
   };
 
-  ::createStackLayout(
+  ::calculateStack(
```

Comment:
By the way, here we actually do not need global scope operator `::`, as the call is not ambiguous.